channel/channel.go

```diff
@@ -154,15 +162,18 @@ func (lc TwoPartyLedger) TheirDestination() types.Destination {
 }
 
 // Clone returns a deep copy of the receiver
-func (c Channel) Clone() Channel {
-	clonedSignedStateForTurnNum := map[uint64]state.SignedState{}
+func (c *Channel) Clone() *Channel {
+	if c == nil {
+		return nil
+	}
+	d, _ := New(c.PreFundState().Clone(), c.MyIndex)
```

Comment:
I think we're ignoring the error here because we know `c.PreFundState().Clone()` starts with turnNum=0 and so we'd never get an error back from `New`?  We could add a comment explaining that.